src/js/components/tour/dropdown.tsx

```diff
@@ -0,0 +1,117 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Checkbox from '@salesforce/design-system-react/components/checkbox';
+import Popover from '@salesforce/design-system-react/components/popover';
+import classnames from 'classnames';
+import i18n from 'i18next';
+import React, { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+import { match as Match, useHistory, useRouteMatch } from 'react-router-dom';
+
+import backpackIcon from '~img/backpack-sm.svg';
+import mapIcon from '~img/map-sm.svg';
+import seesawIcon from '~img/seesaw-sm.svg';
+import { AppState } from '~js/store';
+import { selectProject } from '~js/store/projects/selectors';
+import {
+  SHOW_WALKTHROUGH,
+  WALKTHROUGH_TYPES,
+  WalkthroughType,
+} from '~js/utils/constants';
+import routes, { routePatterns } from '~js/utils/routes';
+
+const TourDropdown = ({ isAlert }: { isAlert?: boolean }) => {
+  const history = useHistory();
+  const match =
+    useRouteMatch<{
+      projectSlug?: string;
+    }>(routePatterns.project_detail()) ||
+    ({ params: {} } as Match<{ projectSlug?: string }>);
+  const selectProjectWithProps = useCallback(selectProject, []);
+  const project = useSelector((state: AppState) =>
+    selectProjectWithProps(state, { match }),
+  );
+  const projectUrl = project ? routes.project_detail(project.slug) : null;
+
+  const handleSelect = useCallback(
+    (type: WalkthroughType) => {
+      /* istanbul ignore else */
+      if (projectUrl) {
+        history.push(projectUrl, { [SHOW_WALKTHROUGH]: type });
+      }
+    },
+    [projectUrl], // eslint-disable-line react-hooks/exhaustive-deps
+  );
+
+  return project ? (
+    <div className="slds-col_padded">
+      <Popover
+        align="bottom right"
+        className={classnames('slds-popover_small', {
+          'slds-text-align_left': isAlert,
+          'slds-text-color_default': isAlert,
```

Comment:
Ugh. We have to override some styles when this dropdown is placed within the Alert, if we want it to look consistent.